scripts/build.py

```diff
@@ -97,7 +97,7 @@
 # /extensions.)
 JS_FILENAME_SUFFIXES_TO_IGNORE = ('Spec.js', 'protractor.js')
 JS_FILENAME_SUFFIXES_NOT_TO_MINIFY = ('.bundle.js',)
-GENERAL_FILENAMES_TO_IGNORE = ('.pyc', '.stylelintrc')
+GENERAL_FILENAMES_TO_IGNORE = ('.pyc', '.stylelintrc', '.DS_Store')
```

Comment:
Will this fix situations where there is a suffix after .DS_Store?

Reply:
There shouldn't be any suffix after `.DS_Store` from what I have read.

Reply:
ah, okay